src/fragments/lib/push-notifications/react-native/getting_started/60_init_push_notifications.mdx

```diff
@@ -0,0 +1,41 @@
+To initialize the Amplify Push Notifications, you will first need to configure Amplify.
+
+Import and load the configuration file in your app. It's recommended you add the Amplify configuration step to your app's root entry point. For example, `index.js`
+
+```js
+// Example index.js
+...
+import { Amplify, Notifications } from 'aws-amplify';
+import awsconfig from './src/aws-exports';
+
+Amplify.configure(awsconfig);
+...
+```
+
+Next, enable Amplify Push Notifications. It is important that you do this in your app's root entry point as push notifications can occur even while your app is in a terminated state.
+
+```js
+// Example index.js
+...
+Notifications.Push.enable();
```

Comment:
do I need to do this before or after the configure function. Does it matter?

Reply:
I think it should take place after. I will update the wording a bit to reflect that